models/arrakis/optimism/arrakis_optimism_schema.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+models:
+  - name: arrakis_optimism_uniswap_pools
```

Comment:
add unique combo of columns to this incremental model

Reply:
adding